test/unit/cron-jobs/removeUnverifiedUsers.spec.js

```diff
@@ -0,0 +1,68 @@
+const tokenService = require('~/services/token')
+const userService = require('~/services/user')
+const {removeUsersWithUnconfirmedEmail} = require('~/cron-jobs/removeUnverifiedUsers')
+
+const mockedUsersList = [{
+  user:{
+    isEmailConfirmed:false
+  },
+  confirmToken:'78fdh78fsd78h'
+}]
+
+jest.mock('~/services/token',()=>({
+  findTokensWithUsersByParams: jest.fn(() => mockedUsersList),
+  validateConfirmToken: jest.fn(),
+  findToken: jest.fn(() => (
+    {
+      userId:'someId',
+    }))
+    ,
+    removeConfirmToken: jest.fn()
+}))
+jest.mock('~/services/user',()=>({
+  deleteUser: jest.fn()
+
+}))
+
+describe('removeUsersWithUnconfirmedEmail', () => {
+    it('should removes unconfirmed users successfully', async () => {
+      await removeUsersWithUnconfirmedEmail()
+
+      expect(userService.deleteUser).toHaveBeenCalledWith('someId')
```

Comment:
Please move all the values, like `someId` here, into constants and then reuse them, to not use hard-coded values.

Reply:
I think it is redundant because i can’t see cases where we can reuse it now

Reply:
You reuse it in your code